its/ruling/src/test/resources/expected_extended/scikit-learn/python-S5644.json

```diff
@@ -1,6 +1,14 @@
 {
```

Comment:
When encountering issue differences on ruling, can you clarify whether it's TP/FP and what is your overall evaluation of the differences?

Given these are on sklearn itself, I would assume these are FPs due to some mismatch between the sklearn version analyzed and the version of the stubs we use? 
I know we have [SONARPY-1518](https://sonarsource.atlassian.net/browse/SONARPY-1518) to eventually deal with this, but another reviewer may not know that.